docs/specs/learn.yml

```diff
@@ -1575,3 +1575,96 @@ outputs:
         }
       }
     }
+---
+# Validate if jupyter notebook points to insecure link
+noSingleFile: true
```

Comment:
Why `noSingleFile`, this doesn't sound it requires a full build

Reply:
@yufeih , you are correct. No full build is required here, I just misunderstood this flag. Will remove this flag from tests. Btw, if any documentation/comment to indicate the meaning for DocfxTestSpec would be appreciated.